packages/keycloak-user-management/src/components/UserList/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -4,4 +4,6 @@ exports[`components/UserList renders user list correctly: header actions col pro
 
 exports[`components/UserList renders user list correctly: table text 1`] = `"First NameLast NameUsernameActionsBenjaminMwalimumwalimuEditOnakenyaonaEditOnaAdminona-adminEditDemokenyaopensrpEdit"`;
 
+exports[`components/UserList search works correctly: full rendered text 1`] = `"User ManagementAdd UserFirst NameLast NameUsernameActionsBenjaminMwalimumwalimuEditOnakenyaonaEditOnaAdminona-adminEditDemokenyaopensrpEdit120 / page"`;
```

Comment:
please confirm that this snapshot was intentionally written, I fail to see a corresponding change that would have led to creation of it.